spec/requests/users_spec.rb

```diff
@@ -1,35 +1,35 @@
-require 'rails_helper'
+# require 'rails_helper'
 
-RSpec.describe 'Users', type: :request do
-  describe 'GET /index' do
-    before(:example) { get '/users' }
+# RSpec.describe 'Users', type: :request do
+#   describe 'GET /index' do
+#     before(:example) { get '/users' }
 
-    it 'Correct Status' do
-      expect(response).to have_http_status(:ok)
-    end
+#     it 'Correct Status' do
+#       expect(response).to have_http_status(:ok)
+#     end
 
-    it 'Correct template' do
-      expect(response).to render_template(:index)
-    end
+#     it 'Correct template' do
+#       expect(response).to render_template(:index)
+#     end
 
-    it 'Correct Content' do
-      expect(response.body).to include('Index')
-    end
-  end
+#     it 'Correct Content' do
+#       expect(response.body).to include('Index')
+#     end
+#   end
 
-  describe 'GET /show' do
-    before(:example) { get '/users/:id' }
+#   describe 'GET /show' do
+#     before(:example) { get '/users/:id' }
 
-    it 'Correct Status' do
-      expect(response).to have_http_status(:ok)
-    end
+#     it 'Correct Status' do
+#       expect(response).to have_http_status(:ok)
+#     end
 
-    it 'Correct template' do
-      expect(response).to render_template(:show)
-    end
+#     it 'Correct template' do
+#       expect(response).to render_template(:show)
+#     end
 
-    it 'Correct Content' do
-      expect(response.body).to include('Show')
-    end
-  end
-end
+#     it 'Correct Content' do
+#       expect(response.body).to include('Show')
+#     end
+#   end
+# end
```

Comment:
- [ ] Please kindly delete all commented code from your code base as this makes your code base dirty and less readable.